MetaMask/metamask-extension
# fixed button in wallet-overview

This PR is to update the portfolio button label for different languages

Fixes #19921 

## Screenshots


### Before
![Screenshot 2023-07-07 at 6 27 22 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/a907eaeb-8a18-44d2-a172-7dfe395ff08a)


### After
![Screenshot 2023-07-07 at 6 27 05 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/7d2d9e6e-11cf-4251-8928-23aad11c9268)


## Manual Testing Steps

- Switch to multiple languages
- Go to home screen
- Check the button text doesn't overflow in wallet overview


## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
